src/enums/entitySubtypes.ts

```diff
@@ -0,0 +1,12 @@
+/** Entity subtype for Coop IA. */
+export enum CoopType {
+  COMMUNITY_SERVICE_COOPERATIVE = 'CSC',
+  HOUSING_COOPERATIVE = 'HC',
+  ORDINARY_COOPERATIVE = 'CP',
+}
+
+/** Entity subtype for Sole Prop registration. */
+export enum BusinessType {
+  DBA = 'DBA',
+  SP = 'SP',
```

Comment:
Interesting! 
I'm surprised we didn't have this before, or did we in some other form?

Reply:
This is a new subtype field in the filing. As I understand it, it's only valid for Sole Prop filings. (Similar to Association Type that is used as a subtype for Coop IAs.)

I added this in after chatting with Vysakh. We will use this soon I think.

Reply:
Here's the new Business Type field in the registration schema:
https://github.com/bcgov/business-schemas/blob/6eee00ec5035b90ab5fb7b74dc5697ec172e721f/src/registry_schemas/schemas/registration.json#L27